frameworks/Go/gnet/benchmark_config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "framework": "gnet",
+  "tests": [{
+    "default": {
+      "plaintext_url": "/plaintext",
+      "port": 8080,
```

Comment:
Can you add a JSON test, too? Do you plan on adding other tests?

Reply:
Actually gnet is like [evio in TechEmpower](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Go/evio/benchmark_config.json), but I guess I could also add JSON test too, I will do that later.

Reply:
Sorry, I now realize that I can't add JSON test cuz the HTTP server based on gnet is just a simplest version without router so it is not capable to distinguish different APIs, hence I would add only plaintext test, please note.

Reply:
Ok, I'm going to merge this for now because evio is doing the same thing, but we may want to consider a different classification for these types of frameworks in the future.

Reply:
Sure, feel free to reach me if anything I could help.